modules/bootkube/resources/manifests/kube-proxy.yaml

```diff
@@ -52,13 +55,12 @@ spec:
         operator: "Exists"
         effect: "NoSchedule"
       volumes:
-      - hostPath:
+      - name: lib-modules
+        hostPath:
+          path: /lib/modules
+      - name: ssl-certs-host
+        hostPath:
           path: /etc/ssl/certs
-        name: ssl-certs-host
       - name: etc-kubernetes
         hostPath:
           path: /etc/kubernetes
-  updateStrategy:
```

Comment:
We want this still here.

Reply:
Is still there, is in a different place please have a look: https://github.com/conde-nast-international/tectonic-installer/blob/b304abaeb0d142799ba57eb094bcddb2f708defd/modules/bootkube/resources/manifests/kube-proxy.yaml#L15-L18

Reply:
duh alright then